vnext/ReactWindowsCore/tracing/fbsystrace.h

```diff
@@ -97,15 +106,17 @@ class FbSystraceSection {
 
   std::string profile_name_;
   uint8_t index_{0};
-  
-  std::chrono::high_resolution_clock::time_point start_{std::chrono::high_resolution_clock::now()};
+
+  std::chrono::high_resolution_clock::time_point start_{
```

Comment:
Not directly related to this CR, but as a warning, high_resolution_clock can have unexpected behavior, like going backwards with things like daylights savings time. steady_clock won't do this, and is still implemented with QueryPerformanceCounter on Windows so you should get a high resolution.